clutter/dna/dna.json

```diff
@@ -0,0 +1,58 @@
+{
+    "Version": 1,
+    "UUID": "ffdcba75-5e0d-11e8-a68e-8c85908a3f13",
+    "Name": "clutter",
+    "Properties": {
+        "description": "provides an application template",
+        "language": "en"
+    },
+    "PropertiesSchemaFile": "properties_schema.json",
+    "BasedOn": {
+        "H": null
+    },
```

Comment:
Old syntax for "BasedOn" which does not work with latest Holochain version.
Just remove this property.